Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -147,18 +167,30 @@ updateSeats()
               ])
           ],
           [
-            "You can add additional team members at anytime from your account page."
+            """
+You must have at least two seats for your team subscription. You can add additional team members at anytime
+from your account page.
+"""
           ]
         )
       ]
     )
   ]
 }
 
-private func teamOwner(_ currentUser: User) -> Node {
+private func teamOwner(currentUser: User, subscribeData: SubscribeConfirmationData) -> Node {
+  guard subscribeData.isOwnerTakingSeat else {
+    return input([
+      name(SubscribeData.CodingKeys.isOwnerTakingSeat.rawValue),
+      `type`(.hidden),
+      value("false")
+      ])
+  }
+
   return gridColumn(
     sizes: [.mobile: 12],
     [
+      id("team-owner"),
```

Comment:
not ideal to add more ids to the page, but I needed this so that I could check if the owner removed themselves from javascript.

an alternative is to look up the input element `input[name=isOwnerTakingSeat]`. thoughts on approaches?

Reply:
I think id is just fine. I'll think on other approaches...but really revamping the way we do JS seems like a larger issue.